### PR TITLE
Fix missing note about debug.dump_destination

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | #5275 |

### CODE REVIEW

**Comment by WouterJ** [comment]
Fixes https://github.com/symfony/symfony-docs/issues/5275

File: components/var_dumper/introduction.rst [hunk 1]
```diff
@@ -111,6 +116,7 @@ original value. You can configure the limits in terms of:
         debug:
            max_items: 250
            max_string_length: -1
+           dump_destination: null
```
Comment (WouterJ) [inline components/var_dumper/introduction.rst]: we always use `~` in Yaml

**Comment by WouterJ** [comment]
Don't we have the PHP config format? And I believe this should be added to the config reference too.

File: components/var_dumper/introduction.rst [hunk 2]
```diff
@@ -104,6 +104,11 @@ original value. You can configure the limits in terms of:
 * maximum number of items to dump,
 * maximum string length before truncation.
 
+Since dumping in the toolbar is not always possible - e.g. when working on a
+JSON API - you can have an alternate output destination for dumps. This is
+configurable with the ``debug.dump_destination``, that you can typically set to
```
Comment (javiereguiluz) [inline components/var_dumper/introduction.rst]: We should probably add the `option` word after `debug.dump_destination`

**Comment by nicolas-grekas** [comment]
Comments addressed

File: components/var_dumper/introduction.rst [hunk 3]
```diff
@@ -104,6 +104,11 @@ original value. You can configure the limits in terms of:
 * maximum number of items to dump,
 * maximum string length before truncation.
 
+Since dumping in the toolbar is not always possible - e.g. when working on a
```
Comment (xabbuh) [inline components/var_dumper/introduction.rst]: dumping *into* the toolbar?

**Comment by nicolas-grekas** [comment]
Comments addressed

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
Thank you for your patience with this PR! Merged now.

**Pull request merged**